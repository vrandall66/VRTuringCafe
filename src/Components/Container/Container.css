.Container {
  display: flex;
  height: 100%;
  flex-wrap: wrap;
  width: 100%;
}