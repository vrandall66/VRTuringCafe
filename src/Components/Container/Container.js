import React from "react";
import Card from "../Card/Card";
import "./Container.css";

const Container = ({ reservations, deleteReservation }) => {
  const allReservations = reservations.map(reservation => {
    return (
      <Card
        reservation={reservation}
        deleteReservation={deleteReservation}
        key={reservation.id}
      />
    );
  });
  return (
    <>
      <div className="Container">{allReservations}</div>
    </>
  );
};

export default Container;
