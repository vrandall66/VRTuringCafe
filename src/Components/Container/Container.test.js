import React from 'react';
import { shallow } from 'enzyme';
import Container from './Container'

describe("Container", () => {
  let wrapper;
  let reservations = [
    {
      name: "",
      date: "",
      number: 0,
      time: "",
      id: 4
    },
    {
      name: "",
      date: "",
      number: 1,
      time: "",
      id: 5
    }
  ]
  
  beforeEach(() => {
    wrapper = shallow(<Container reservations={reservations} />)
  })

  it("should match the snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  })
})