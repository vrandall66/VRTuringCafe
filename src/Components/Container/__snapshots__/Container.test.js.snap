// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container should match the snapshot 1`] = `
<Fragment>
  <div
    className="Container"
  >
    <Card
      key="4"
      reservation={
        Object {
          "date": "",
          "id": 4,
          "name": "",
          "number": 0,
          "time": "",
        }
      }
    />
    <Card
      key="5"
      reservation={
        Object {
          "date": "",
          "id": 5,
          "name": "",
          "number": 1,
          "time": "",
        }
      }
    />
  </div>
</Fragment>
`;
